sys/net/gnrc/link_layer/lwmac/lwmac.c

```diff
@@ -64,6 +64,38 @@ static void lwmac_set_state(gnrc_netdev_t *gnrc_netdev, gnrc_lwmac_state_t newst
 static void lwmac_schedule_update(gnrc_netdev_t *gnrc_netdev);
 static void rtt_handler(uint32_t event, gnrc_netdev_t *gnrc_netdev);
 
+static void lwmac_reinit_radio(gnrc_netdev_t *gnrc_netdev)
```

Comment:
So, a `lwmac_init_radio` function doesn't exist? If so, I'd prefer to rename it that way. I'd say this function should be called when the rdc starts to ensure a good starting point.